nimble/include/nimble/nimble_npl.h

```diff
@@ -68,8 +68,13 @@ struct ble_npl_eventq *ble_npl_eventq_dflt_get(void);
 
 void ble_npl_eventq_init(struct ble_npl_eventq *evq);
 
+struct ble_npl_event *ble_npl_eventq_get_tmo(struct ble_npl_eventq *evq,
+                                             ble_npl_time_t tmo);
+
 struct ble_npl_event *ble_npl_eventq_get(struct ble_npl_eventq *evq);
 
+struct ble_npl_event *ble_npl_eventq_get_no_wait(struct ble_npl_eventq *evq);
```

Comment:
this can be actually `ble_npl_eventq_get_tmo(evq, 0)` - no need to create too many functions
(but I'd keep `ble_npl_evqnt_get` for "historical" reasons ;-) )